### PR TITLE
Initial governance model and principles for phosphor

### PR DESCRIPTION
**Pull request by ellisonbg**
@sccolbert @jasongrout @blink1073 @afshin @ian-r-rose @vidartf @saulshanabrook

This is an initial governance model for phosphor. @sccolbert and I have gone over this in a call.

Once this has been reviewed and merged, and an initial group of Package Maintainers are appointed, there are a number of outstanding questions to tackle, that we wanted to get the PM's input on:

- Determine response times for PRs and issues.
- How long to wait for CE vetos?
- Process around CE stepping down, electing a new one?
- How long do we give package maintainers to create initial tutorial documentation?
- How to manage GitHub milestones?
- How many votes are needed for a quorum?
  * Suggestion: 100% expected unless unable due to emergency. Votes are subject to a time limit (tbd). Maintainers missing a vote due to a non-emergency take a demerit.
  * Old language: “All votes described herein shall take place on GitHub (using thumbs up/down) and require all Package Maintainers to vote within X hours. “

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* PhosphorJS is about code, documentation, and technical excellence. Leave
+  non-technical discussions and opinions at the door.
+* Code of Conduct: Be nice.
+
+## Governance Model
+
+* PhosphorJS is governed by a group of Package Maintainers and a Chief Executive,
+  that lead the project.
+* Package Maintainers are responsible for one or more PhosphorJS packages. Each
+  package shall have a one or more maintainers.
```
Comment (afshin) [inline GOVERNANCE.md]: "a one or more maintainers" => "one or more maintainers"
(minor typo)

**Comment by afshin** [comment]
Thank you for writing this, Brian. I don't have any substantive additions or subtractions to make. I think this is a sound basis to work with, as long as others share in the consensus.

**Comment by blink1073** [comment]
Thank you, Brian, these changes look good to me.

**Comment by ellisonbg** [comment]
We can wait for more input, but from my perspective the final decision is for @sccolbert 

I am fine with this being merged whenever you are ready

**Comment by sccolbert** [comment]
Thanks Brian! This is great!

**Pull request merged**